src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html

```diff
@@ -7,5 +7,5 @@
 Consider copying the report to a location outside the workspace and opening it from there. For more information, 
 see <a href="http://wiki.jenkins.io/display/JENKINS/Configuring+Content+Security+Policy">Configuring Jenkins Content Security Policy</a>.
 <br><br>
-<b>Note:</b> This feature is not currently supported on Mac platforms.
+<b>Note:</b> This feature is supported on Mac platforms in MATLAB R2020b and later.
```

Comment:
On macOS platforms, this feature is supported in MATLAB R2020b and later.

Reply:
Done